macros/blueprint-proc-macro/tests/tests.rs

```diff
@@ -1,22 +1,18 @@
+/// Add valid cases under a mod to prove the syntax compiles
+/// We do not do the same for invalid cases since we are proving
+/// there that the syntax does not compile
+pub mod valid_cases;
+
+#[derive(Clone)]
+struct EmptyContext;
+
 #[cfg(test)]
 mod tests {
     use trybuild::TestCases;
 
-    // #[test]
-    // fn test_jobs_valid_cases() {
-    //     let t = TestCases::new();
-    //     t.pass("tests/valid_cases/job/*.rs");
-    // }
-
     #[test]
     fn test_jobs_invalid_cases() {
         let t = TestCases::new();
         t.compile_fail("tests/invalid_cases/job/*.rs");
     }
-
-    #[test]
```

Comment:
The valid test cases you fixed below aren't running now ya? Can we make them run? Or it's simply enough to define them above and ensure they compile?

Reply:
Before, they weren't technically running, only checking for syntactical correctness. Now, it does the same thing but implicitly with being attached to the module tree. We could make them run once to check input/output correctness, though, maybe in another PR since it would be beyond the scope of this task.